# Telomere-to-telomere consortium HG002 project
 
In collaboration with the [Human Pangenome Reference Consortium](https://humanpangenome.org) and the [Genome in a Bottle Consortium](https://www.nist.gov/programs-projects/genome-bottle) we have sequenced and assembled the [HG002](https://www.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=GM24385) aka GM24385 aka huAA53E0 cell line. 
The ultimate goal of this effort is to create a reference assembly for the HG002 reference material that is perfectly accurate.

The previously generated HiFi data is available from the HPRC and ONT data is available from the HPRC and GIAB. The assembly used [verkko](https://github.com/marbl/verkko) with manual assignment of nodes to chromosomes, ONT-based patching to resolve HiFi coverage gaps, manual resolution of tangles, and Strand-Seq and Hi-C based assignment of acrocentric short arms to chromosomes. We have made the assembly available to the community immediately. Although the latest assembly release is gapless (with the excption of the rDNA arrays), it has not been validated and may contain errors, both SNP and SV, and should be treated as a draft. Work on validation and polishing is underway.
 
## Data reuse and license
All assembly data is released to the public domain ([CC0](https://creativecommons.org/publicdomain/zero/1.0/)) and we encourage its reuse. We would appreciate if you would acknowledge and cite the "Telomere-to-Telomere" (T2T) Consortium for the creation of this data. More information about our consortium can be found on the [T2T homepage](https://sites.google.com/ucsc.edu/t2tworkinggroup/) and a list of related citations is available below:
 
### Relevant citations:
1. Rautiainen M, et al. [Verkko: telomere-to-telomere assembly of diploid chromosomes](https://doi.org/10.1101/2022.06.24.497523). BioRxiv, 2022.
2. Wang T. [The Human Pangenome Project: a global resource to map genomic diversity.](https://www.nature.com/articles/s41586-022-04601-8). Nature, 2022.
3. Zook J. et al. [A robust benchmark for detection of germline large deletions and insertions](https://doi.org/10.1038/s41587-020-0538-8). Nature Biotech, 2022.
 
# Assembly releases
### v0.7
<a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/scratch/HG002/assemblies/drafts/assembly.v0.7.fasta">T2T reconstruction of all 46 chromosomes of HG002</a> along with mitochondria and EBV sequences. Maternal and paternal haplotypes are denoted by contig name. Unassigned contigs are from unresolved rDNA arrays. The pipeline used two versions of verkko combined with manual curation and resolution. Assemblies using both versions are available below along with the final traversals used for each chromosome and haplotype. Negative gaps indicate there was an overlap between the verkko contigs, supported by ONT-only assembly and sequences. Note that additional files, including ONT coverage and trio coloring, are available for each assembly. You can browse these from the top-level assembly [folder](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/assemblies/graphs/).
 
Consensus quality exceeds Q60. Centers of 9/10 rDNA arrays are represented by N-gaps.
 
## Downloads
- <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/scratch/HG002/assemblies/drafts/assembly.v0.7.fasta">Assembly draft v0.7</a> (md5: c5ecb037015955ffa44b76eb674923f9 )
- <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/scratch/HG002/assemblies/graphs/20220725/assembly.homopolymer-compressed.noseq.gfa">Assembly graph from Verkko v1.1 in GFA format, no sequences included (md5: a7473ba67b1109089217ee4b7fdf647a )
- <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/scratch/HG002/assemblies/graphs/20220812/assembly.homopolymer-compressed.noseq.gfa">Assembly graph from Verkko v1.2/MBG v1.0.13, no sequences included (md5: 86f896e5232a5aee95b47f32fbddd114)
- <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/scratch/HG002/assemblies/drafts/v0.7/assembly.v0.7.chromosome_paths.gaf">Genomic paths through both graphs used for consensus. ChrX and chr12 maternal were previously finished and are excluded </a> (md5: bf07da577141d4240f54b18091f0b35d)
 
# Sequencing Data
 
## HiFi Data
Sequenced used for the assembly were generated by HPRC and GIAB and are available from [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/hifi/).
 
## Oxford Nanopore Data
Nanopore sequencing was performed by HPRC and GIAB. The data used for the assembly are available from [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/ont/).
 
## StrandSeq
Strandseq data is available from the [HPRC](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=working/HPRC_PLUS/HG002/raw_data/Strand_seq/)
 
## HiC
Hi-C data is available from the [HPRC](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=working/HPRC_PLUS/HG002/raw_data/hic/downsampled/). The assembly used HG002.HiC_2_NovaSeq_rep1_run2_S1_L001_R1_001.fastq.gz and HG002.HiC_2_NovaSeq_rep1_run2_S1_L001_R2_001.fastq.gz.
 
# Notes on downloading files.
 
Files are generously hosted by Amazon Web Services. Although available as straight-forward HTTP links, download performance is improved by using the Amazon Web Services <a href="https://aws.amazon.com/cli/">command-line interface</a>. References should be amended to use the `s3://` addressing scheme, i.e. replace `https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/` with `s3://human-pangenomics/T2T` to download.
 
The s3 command can also be used to get information on the dataset, for example reporting the size of every file in human-readable format.
 
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/scratch/HG002/
 
or to obtain technology-specific sizes.
 
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/scratch/HG002/sequencing
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/scratch/HG002/assemblies
 
Amending the `max_concurrent_requests` etc. settings as per <a href="https://docs.aws.amazon.com/cli/latest/topic/s3-config.html">this guide</a> will improve download performance further.
 
You can also browse all the files available on S3 via <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/HG002/">web interface</a>.
 
# Contact
 
Please raise issues on this Github repository concerning this dataset.
 
# History
 
    * v0.7: 7th November 2022. Initial release.
